contrib/inventory/azure_rm.py

```diff
@@ -107,6 +107,11 @@
   "virtual_machine_size": "Standard_DS4"
 }
 
+Run for Specific Virtual machine scale set
```

Comment:
I guess for consistency everything should be uppercase "Specific Virtual Machine Scaleset". Should "scaleset" be one word or two words? "scale set"

Reply:
Ok, I checked, It should be "scale set"

Reply:
fixed